contracts/FeeBurner.sol

```diff
@@ -109,7 +109,7 @@ contract FeeBurner is Withdrawable, FeeBurnerInterface, Utils2 {
         require(msg.sender == address(kyberNetwork));
         require(tradeWeiAmount <= MAX_QTY);
 
-        uint kncAmount = calcDestAmount(ETH_TOKEN_ADDRESS, ERC20(knc), tradeWeiAmount, kncPerEthRatePrecision);
+        uint kncAmount = calcDestAmountWithDecimals(ETH_DECIMALS, MAX_DECIMALS, tradeWeiAmount, kncPerEthRatePrecision);
```

Comment:
lets have a comment explaining that `MAX_DECIMAL` equals to decimal of KNC

Reply:
PR to add the comment here: https://github.com/KyberNetwork/smart-contracts/pull/489